CHANGES.rst

```diff
@@ -12,10 +12,18 @@ sbpy.data
 
 - Added ``Orbit.D_criterion`` to evaluate the D-criterion between two sets
   of orbital elements. [#325]
+
 - Added ``DataClass.__contains__`` to enable `in` operator for ``DataClass``
   objects. [#357]
 
 
+sbpy.photometry
+^^^^^^^^^^^^^^^
+
+- Added parameter constraints to the IAU disk-integrated phase function models,
+  such as ``HG``, ``HG1G2``, ``HG12``, and ``HG12_Pen16``. [#366]
+
```

Comment:
Also, note change API: the NonmonotonicPhaseFunctionWarning has been replaced with InvalidPhaseFunctionWarning